charts/eol-exporter/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+name: eol-exporter
+description: |
+  End of life exporter.
+  A Kubernetes's helm chart for a exporter that get information about end of life/support of products in order to be scrapped by Prometheus
+
+  You must supply a valid configmap with a list of products with its versions. Check https://github.com/saritasa-nest/saritasa-devops-tools-eol-exporter/blob/main/config.yaml.example
+  for example values.
+  Each product must have a field `current` with valid version as defined in: https://endoflife.date/api/{product}.json
+  A `comment` field is optional, and it will be added as a label in the metrics.
```

Comment:
since this is a public chart and the mentioned repo is private - pls post here actual configuration sample

Reply:
Done